### PR TITLE
Adding 60-day rolling volatility information

### PR DESCRIPTION
**Pull request by mecampbellsoup**

### CODE REVIEW

File: code/update.go [hunk 1]
```diff
@@ -64,9 +65,19 @@ func updateHandler(w http.ResponseWriter, r *http.Request) {
 		for _, point := range subset {
 			returns = append(returns, point.Return)
 		}
-		data[j].Volatility = stats.StatsSampleStandardDeviation(returns) * 100.0
+		data[j].Volatility30 = stats.StatsSampleStandardDeviation(returns) * 100.0
 		j += 1
 	}
+	k := 59
```
Comment (mecampbellsoup) [inline code/update.go]: @elidourado This is really the only section I'm not 100% on...

**Comment by elidourado** [comment]
Hey, thanks for the PR. From reading the changes, this looks like it will work to me. One thing that still needs to be done is to make it not break the API by changing the names of returned parameters, for example at https://btcvol.info/latest. I think one way to do this is to revert "Volatility30" everywhere back to "Volatility."

Would you mind also submitting a screenshot of how the chart looks with the multiple series?

**Comment by mecampbellsoup** [comment]
Nice! Sure thing, will get back to this within the next day or so :smile:

**Comment by mecampbellsoup** [comment]
Can you elaborate on "not break the API by changing the names of returned parameters"? Is this more of a 'best practice for APIs' point you're making or am I missing your point?

**Comment by elidourado** [comment]
My only concern is that some people are already using the API. They retrieve the data at /latest which currently returns a JSON object with a Date element and a Volatility element. Your pull request changes the name of that Volatility element to Volatility30. So when their code looks for Volatility, it won't exist.

In order to prevent people's code from breaking, we need Volatility to remain in the JSON object returned by the API. It shouldn't be a problem to add Volatility60 to the data returned, but we shouldn't just change a name of an element that people are relying on because that may case errors for people using the API.

**Comment by mecampbellsoup** [comment]
@elidourado OK, unbroke the API name change - should be good to go! :smiley:

**Pull request merged**